src/main/scala/com/gu/fezziwig/CirceScroogeMacros.scala

```diff
@@ -238,17 +238,17 @@ private class CirceScroogeMacrosImpl(val c: blackbox.Context) {
     q"""{
       new _root_.io.circe.Decoder[$A] {
         def apply(c: _root_.io.circe.HCursor): _root_.io.circe.Decoder.Result[$A] = {
-          val result = c.fields.getOrElse(Nil).headOption.flatMap {
+          val maybeResult = c.fields.getOrElse(Vector.empty).headOption.flatMap {
```

Comment:
> encoding the type in the variable name hungarian notation is usually not recommended in scala

I personally find `maybeResult` for an optional value much better for readability than just result. The alternative if not preferred it to always explicitly declare the type.